core/templates/pages/contributor-dashboard-page/opportunities-list/opportunities-list.component.ts

```diff
@@ -92,6 +92,7 @@ export class OpportunitiesListComponent {
   }
 
   ngOnInit(): void {
+    this.loadingOpportunityData = true;
```

Comment:
Why do we need to do this if `loadingOpportunityData` is set to `true` on initialization?

Reply:
So as you can see in screenshots below, when a Language Change or Topic Change happens, the code runs `ngOnInit()` and loads new opportunities. 

However, `ngOnInit()`never sets the value `loadingOpportunityData` to `true`, and because of that  `loadingOpportunityData` is always `false` and the`opportunity-loading-placeholder `will be hidden even when we are loading new opportunities in `ngOnInit()`

![Screen Shot 2021-12-03 at 12 47 08 PM](https://user-images.githubusercontent.com/35385366/144670871-3c051b47-98c5-4d8f-8b95-30004f36ee23.png)

![Screen Shot 2021-12-03 at 12 58 11 PM](https://user-images.githubusercontent.com/35385366/144671996-8f38c9f3-35ac-473b-8852-80b5dddfdef6.png)

Reply:
Ah, gotcha. That clears it up, thanks!